style.css

```diff
@@ -23,3 +22,49 @@ header {
   letter-spacing: 0;
   text-align: center;
 }
+
+.headline {
+  background-image: url("./images/backgroundImage.png");
+  height: calc(100vh - 75px);
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] I have noticed that the background image height is too long :persevere: . Kindly make it professional by setting the reasonable height of the `.headline` :+1: